--- /dev/null
+++ b/BUILD_DIR/fix-x11-conda.sh
@@ -0,0 +1,21 @@
+#!/bin/bash
+PREFIX=$1
+ostype=$(uname)
+if [ "$ostype" = "Darwin" ]; then
+    # use headers from libx11 not the ones shipped with tk
+    # with this, xserver works on mac, but not tkpgplot
+    # This can be done at the user end by: conda install xorg-libx11 --clobber
+    find $PREFIX/include/X11 -name "*.h__clobber-from-xorg-libx11*" \
+        -exec sh -c 'mv "$0" "${0%%__clobber-from-xorg-libx11}"' {} \;
+    find $PREFIX/include/X11 -name "*.h__clobber-from-xorg-xorgproto*" \
+        -exec sh -c 'mv "$0" "${0%%__clobber-from-xorg-xorgproto}"' {} \;
+    
+    # osx-64 does it differently
+    if [ -d $PREFIX/__clobbers__/xorg-libx11 ]; then
+        cp $PREFIX/__clobbers__/xorg-libx11/include/X11/* $PREFIX/include/X11/
+    fi
+    if [ -d $PREFIX/__clobbers__/xorg-xorgproto ]; then
+        cp $PREFIX/__clobbers__/xorg-xorgproto/include/X11/* $PREFIX/include/X11/
+    fi
+fi
+
