--- hitomixrism/common/mission/tasklib/trendtemp/src/Makefile
+++ hitomixrism/common/mission/tasklib/trendtemp/src/Makefile
@@ -8,7 +8,7 @@
 
 HD_CXXFLAGS		= ${HD_STD_CXXFLAGS}
 
-HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahcaldb -lahapp -l${AHFITS} -lahmath -l${AHGEN} -l${AHLOG} \
+HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahcaldb -lahapp -l${AHFITS} -lahmath -lahmission -l${AHGEN} -l${AHLOG} \
 			  ${HD_STD_LIBS} ${F77LIBS4C}
 
 HD_INSTALL_LIBRARIES	= ${HD_LIBRARY_ROOT}
