diff --git a/ftools/asca/tasks/mkfilter2/duvmif2.c b/ftools/asca/tasks/mkfilter2/duvmif2.c
index 255b111334..78de780ce5 100644
--- a/ftools/asca/tasks/mkfilter2/duvmif2.c
+++ b/ftools/asca/tasks/mkfilter2/duvmif2.c
@@ -70,12 +70,10 @@
 #include "cfortran.h"
 #include "cftools.h"
 
-static double	golden();
+static double golden(double, double, double, double (*f)(), double, double *, int *);
 
 int
-duvmif2(f, xguess, step, bound, xacc, maxfn, x)
-     double	(*f)(), *xguess, *step, *bound, *xacc, *x;
-     int	*maxfn;
+duvmif2(double (*f)(double*), double *xguess, double *step, double *bound, double *xacc, int *maxfn, double *x)
 {
   double   a, b, tol, temp1, temp2, bestguess=0.0;
 
@@ -109,8 +107,9 @@ duvmif2(f, xguess, step, bound, xacc, maxfn, x)
   return(0);
 }
 
-FCALLSCSUB7(duvmif2,DUVMIF2,duvmif2,PDOUBLE,PDOUBLE,PDOUBLE,PDOUBLE,PDOUBLE,PINT,PDOUBLE)
-
+#undef ROUTINE_1
+#define ROUTINE_1 (double (*)(double *))
+FCALLSCSUB7(duvmif2, DUVMIF2, duvmif2, ROUTINE, PDOUBLE, PDOUBLE, PDOUBLE, PDOUBLE, PINT, PDOUBLE)
 
 /* Copyright (C) 1987,1988 Numerical Recipes Software -- GOLDEN */
 /* Modified for DOUBLE PRECISION data types */
