diff --git a/ftools/asca/tasks/Makefile b/ftools/asca/tasks/Makefile
index 4b00e18698..b9627b4366 100644
--- a/ftools/asca/tasks/Makefile
+++ b/ftools/asca/tasks/Makefile
@@ -5,7 +5,7 @@ HD_COMPONENT_VERS	=
 HD_SUBDIRS		= ascaarf ascaeffmap ascaexpo ascalin ascaray \
 			  attitude cleansis corpileup correctrdd dfe2mkf faint \
 			  faintdfe fasttime fastwmap fmosaic ghkcurve ghkdump \
-			  gisrti gqaplot ldeadtime mkdtime mkfilter2 mkgisbgd \
+			  gisrti gqaplot ldeadtime mkdtime mkgisbgd \
 			  oldaspect sisclean sishist sispi sqaplot \
 			  sisrmg temp2gain timeconv \
 			  add_sis_rmf addascaspec ascascreen \
