--- a/tcltk/BUILD_DIR/Makefile.in
+++ b/tcltk/BUILD_DIR/Makefile.in
@@ -66,9 +66,6 @@ build-tcl:
 		rm -f ${TCL_DIR}/tclsh${BSUF}; \
 	fi
 	${MAKE} subdir-tcl
-	@if [ "x${EXT}" = xdarwin -a -f ${TCL_DIR}/tclsh${BSUF} ]; then \
-		/usr/bin/install_name_tool -add_rpath ${HD_BLD_LIB} ${TCL_DIR}/tclsh${BSUF}; \
-	fi
 	@if [ "x${SYMBOLS}" = xyes ]; then \
 		${HD_INSTALL} ${TCL_DIR}/lib${TCL_VERS_SHORT}g${SHLIB_SUFFIX} \
 			${TCL_DIR}/lib${TCL_VERS_SHORT}${SHLIB_SUFFIX} ${HD_LN_S}; \
@@ -114,11 +111,7 @@ all-tk default-tk: build-tk publish-tk
 
 build-tk:
 	@rm -f ${TK_DIR}/wish${BSUF}
-	${MAKE} subdir-tk
-	@if [ "x${EXT}" = xdarwin -a -f ${TK_DIR}/wish${BSUF} ]; then \
-		echo "/usr/bin/install_name_tool -add_rpath ${HD_TOP_LIB} ${TK_DIR}/wish${BSUF}"; \
-		/usr/bin/install_name_tool -add_rpath ${HD_TOP_LIB} ${TK_DIR}/wish${BSUF}; \
-	fi
+	${MAKE} subdir-tk CFLAGS+="-std=gnu17"
 
 publish-tk:
 	${MAKE} install-tk HD_TOP_EXEC_PFX=${HD_BLD_EXEC_PFX} HD_EXEC_PFX=${HD_BLD_EXEC_PFX} HD_CP_P="${HD_LN_S}"
@@ -186,7 +179,7 @@ build-itk:
 	${MAKE} subdir-itk
 
 publish-itk:
-	:
+	${MAKE} install-itk HD_TOP_EXEC_PFX=${HD_BLD_EXEC_PFX} HD_EXEC_PFX=${HD_BLD_EXEC_PFX} HD_CP_P="${HD_LN_S}"
 
 install-itk:
 	${MAKE} subdir-itk HD_TARGET="install" \
@@ -214,7 +207,7 @@ build-iwidgets:
 	${MAKE} subdir-iwidgets
 
 publish-iwidgets:
-	:
+	${MAKE} install-iwidgets HD_TOP_EXEC_PFX=${HD_BLD_EXEC_PFX} HD_EXEC_PFX=${HD_BLD_EXEC_PFX} HD_CP_P="${HD_LN_S}"
 
 install-iwidgets:
 	${MAKE} subdir-iwidgets HD_TARGET="install" \
@@ -264,7 +257,7 @@ build-tix:
 	${MAKE} subdir-tix
 
 publish-tix:
-	:
+	${MAKE} install-tix HD_TOP_EXEC_PFX=${HD_BLD_EXEC_PFX} HD_EXEC_PFX=${HD_BLD_EXEC_PFX} HD_CP_P="${HD_LN_S}"
 
 install-tix:
 	${MAKE} subdir-tix HD_TARGET="install" \
