--- a/tcltk/tix/configure
+++ b/tcltk/tix/configure
@@ -6645,9 +6645,9 @@ echo $ECHO_N "checking for Tcl private include files... $ECHO_C" >&6
 	        ;;
 	esac
     else
-	if test ! -f "${TCL_SRC_DIR}/generic/tclInt.h" ; then
-	    { { echo "$as_me:$LINENO: error: Cannot find private header tclInt.h in ${TCL_SRC_DIR}" >&5
-echo "$as_me: error: Cannot find private header tclInt.h in ${TCL_SRC_DIR}" >&2;}
+	if test ! -f "${TCL_SRC_DIR}/generic/tclInt.h" -a ! -f "${TCL_SRC_DIR}/tclInt.h"; then
+	    { { echo "$as_me:$LINENO: error: Cannot find private header tclInt.h in ${TCL_SRC_DIR}/generic or ${TCL_SRC_DIR}" >&5
+echo "$as_me: error: Cannot find private header tclInt.h in ${TCL_SRC_DIR}/generic or ${TCL_SRC_DIR}" >&2;}
    { (exit 1); exit 1; }; }
 	fi
     fi
@@ -6700,9 +6700,9 @@ echo $ECHO_N "checking for Tk private include files... $ECHO_C" >&6
 	        ;;
 	esac
     else
-	if test ! -f "${TK_SRC_DIR}/generic/tkInt.h" ; then
-	    { { echo "$as_me:$LINENO: error: Cannot find private header tkInt.h in ${TK_SRC_DIR}" >&5
-echo "$as_me: error: Cannot find private header tkInt.h in ${TK_SRC_DIR}" >&2;}
+	if test ! -f "${TK_SRC_DIR}/generic/tkInt.h" -a ! -f "${TK_SRC_DIR}/tkInt.h"; then
+	    { { echo "$as_me:$LINENO: error: Cannot find private header tkInt.h in ${TK_SRC_DIR}/generic or ${TK_SRC_DIR}" >&5
+echo "$as_me: error: Cannot find private header tkInt.h in ${TK_SRC_DIR}/generic or ${TK_SRC_DIR}" >&2;}
    { (exit 1); exit 1; }; }
 	fi
     fi
