--- Xspec/BUILD_DIR/configure
+++ Xspec/BUILD_DIR/configure
@@ -5866,7 +5866,8 @@
 printf "%s\n" "$ac_cv_cxx_std_count_defect" >&6; }
   if test "$ac_cv_cxx_std_count_defect" = yes; then
 
-printf "%s\n" "#define STD_COUNT_DEFECT 1" >>confdefs.h
+#printf "%s\n" "#define STD_COUNT_DEFECT 1" >>confdefs.h
+echo "NA"
 
   fi
--- a/Xspec/BUILD_DIR/hd_config_info
+++ b/Xspec/BUILD_DIR/hd_config_info
@@ -1,9 +1,12 @@
 COMPONENT=Xspec
 VERSION=
 
-CONFIGURE_CMD='cd $h_dir; ./configure --build=$build --with-heatop="$prefix" --enable-collapse=$h_collapse --with-heacore=$h_config_heacore --enable-shared=$h_shared --enable-symbols=$h_symbols --enable-strip=$h_strip --enable-guis=no --with-copt="$h_copt" --with-cxxopt="$h_cxxopt" --with-fopt="$h_fopt" --enable-ldopt="$h_ldopt" --with-components="$h_Xspec_components" --enable-hera=$h_hera_build --with-png=$h_png_pfx --x-libraries=$x_libraries --x-includes=$x_includes --enable-openmp=$h_openmp --enable-mac_intel_build=$h_mac_intel_build --enable-x=$h_x11 --host=$target'
-
 HEACORE=heacore
 TCLTK=tcltk
 
+HEATOOLS=heatools
+ATTITUDE=attitude
+
+CONFIGURE_CMD='cd $h_dir; ./configure --build=$build --with-heatop="$prefix" --enable-collapse=$h_collapse --with-heacore=$h_config_heacore --with-heatools=$h_config_heatools --enable-shared=$h_shared --enable-symbols=$h_symbols --enable-strip=$h_strip --enable-guis=no --with-copt="$h_copt" --with-cxxopt="$h_cxxopt" --with-fopt="$h_fopt" --enable-ldopt="$h_ldopt" --with-components="$h_Xspec_components" --enable-hera=$h_hera_build --with-png=$h_png_pfx --x-libraries=$x_libraries --x-includes=$x_includes --enable-openmp=$h_openmp --enable-mac_intel_build=$h_mac_intel_build --enable-x=$h_x11 --host=$target'
+
 DEPENDENCIES="$HEACORE $TCLTK"
