--- ftools/guis/fitsTcl/Makefile
+++ ftools/guis/fitsTcl/Makefile
@@ -11,7 +11,7 @@
 
 HD_INSTALL_LIBRARIES	= ${HD_LIBRARY_ROOT}
 
-HD_SHLIB_LIBS		= ${HD_LFLAGS} ${TCL} ${CURL_LIB} ${SYSLIBS}
+HD_SHLIB_LIBS		= ${HD_LFLAGS} ${TCL} -lz ${CURL_LIB} ${SYSLIBS}
 
 HD_INSTALL_HEADERS	= fitsTcl.h fitsTclInt.h
 
--- tcltk/pow/Makefile
+++ tcltk/pow/Makefile
@@ -18,7 +18,7 @@
 
 HD_INSTALL_LIBRARIES	= ${HD_LIBRARY_ROOT}
 
-HD_SHLIB_LIBS		= ${HD_LFLAGS} -l${CFITSIO} ${TCL} ${TK} ${XLIBS} ${CURL_LIB} ${SYSLIBS}
+HD_SHLIB_LIBS		= ${HD_LFLAGS} -l${CFITSIO} ${TCL} ${TK} -lz ${XLIBS} ${CURL_LIB} ${SYSLIBS}
 
 HD_INSTALL_SHELL_SCRIPTS	= POWplot
 
