--- BUILD_DIR/hmake
+++ BUILD_DIR/hmake
@@ -200,7 +200,7 @@
 
 # assemble a path upon which to look for a resource file
 # in any case, first place to look is ., then ./BUILD_DIR, and where this script lives
-hmakerc_path="$this_dir:$this_dir/BUILD_DIR:$scripthome"
+hmakerc_path="$this_dir:$this_dir/BUILD_DIR:$scripthome:$scripthome/../BUILD_DIR:"
 
 # if headaspkg is an empty string, there was no HD_COMPONENT_NAME
 if [ "x$headaspkg" = x ]; then
@@ -212,7 +212,7 @@
   # recurse upwards, looking for BUILD_DIR until we reach the top of this
   # package
   # start one level up from current directory
-  pkg_hmakerc_path=
+  pkg_hmakerc_path="$scripthome/../$headaspkg/BUILD_DIR"
   tmp=`cd $this_dir/..; pwd`
 
   # hmake assumes that while the current directory contains the string
