--- Xspec/src/XSModel/Model/MixFunction/Makefile
+++ Xspec/src/XSModel/Model/MixFunction/Makefile
@@ -39,7 +39,7 @@
 HD_CXXFLAGS		= ${HD_STD_CXXFLAGS} -DINITPACKAGE
 
 HD_SHLIB_LIBS		= ${HD_LFLAGS}  \
-			  -lXSFunctions -lXSModel -lXSUtil -lXS \
+			  -lXSFunctions -lXSModel -lXSUtil -lXS -l${HEASP} \
 			  ${GUI_LIBS} -l${TCLREADLINE} -l${TCL} ${GSL_LIBS} \
 			  ${HD_STD_LIBS} ${XLIBS} ${SYSLIBS} ${F77LIBS4C}
 
