--- hitomixrism/common/cal/tasklib/calanticopi/src/Makefile
+++ hitomixrism/common/cal/tasklib/calanticopi/src/Makefile
@@ -8,7 +8,7 @@
 
 HD_CXXFLAGS		= ${HD_STD_CXXFLAGS}
 
-HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahcaldb -lahapp -l${AHFITS} -lahmath -l${AHGEN} -l${AHLOG} \
+HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahcaldb -lahapp -l${AHFITS} -lahmath -lahmission -l${AHGEN} -l${AHLOG} \
 			  ${HD_STD_LIBS} ${F77LIBS4C}
 
 HD_INSTALL_LIBRARIES	= ${HD_LIBRARY_ROOT}
--- hitomixrism/common/cal/tasklib/calbranch/src/Makefile
+++ hitomixrism/common/cal/tasklib/calbranch/src/Makefile
@@ -8,7 +8,7 @@
 
 HD_CXXFLAGS		= ${HD_STD_CXXFLAGS}
 
-HD_SHLIB_LIBS		= ${HD_LFLAGS} -lrmflib -lahcaldb -larfgenlib -lrmflib -lahapp -l${AHFITS} -lahmath -l${AHGEN} -l${AHLOG} \
+HD_SHLIB_LIBS		= ${HD_LFLAGS} -lrmflib -lahcaldb -larfgenlib -lrmflib -lahapp -lahmission -lsxsrsl -l${AHFITS} -lahmath -l${AHGEN} -l${AHLOG} \
 			  ${HD_STD_LIBS} ${F77LIBS4C}
 
 HD_INSTALL_LIBRARIES	= ${HD_LIBRARY_ROOT}
--- hitomixrism/common/ccd/tasklib/ccdphas/src/Makefile
+++ hitomixrism/common/ccd/tasklib/ccdphas/src/Makefile
@@ -8,7 +8,7 @@
 
 HD_CXXFLAGS		= ${HD_STD_CXXFLAGS}
 
-HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahcaldb -lahapp -l${AHFITS} -lahmath -l${AHGEN} -l${AHLOG} \
+HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahcaldb -lahapp -l${AHFITS} -lahmath -lahmission -l${AHGEN} -l${AHLOG} \
 			  ${HD_STD_LIBS} ${F77LIBS4C}
 
 HD_INSTALL_LIBRARIES	= ${HD_LIBRARY_ROOT}
--- hitomixrism/common/mission/tasklib/expmap/src/Makefile
+++ hitomixrism/common/mission/tasklib/expmap/src/Makefile
@@ -8,7 +8,7 @@
 
 HD_CXXFLAGS		= ${HD_STD_CXXFLAGS}
 
-HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahcaldb -lahapp -l${AHFITS} -lahmath -l${AHGEN} -l${AHLOG} \
+HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahcaldb -lahapp -l${AHFITS} -lahmath -lahmission -larfgenlib -limagetranslib -lcoordfits -lcoord -l${AHGEN} -l${AHLOG} \
 			  ${HD_STD_LIBS} ${F77LIBS4C}
 
 HD_INSTALL_LIBRARIES	= ${HD_LIBRARY_ROOT}
--- hitomixrism/common/mission/tasklib/mktim/src/Makefile
+++ hitomixrism/common/mission/tasklib/mktim/src/Makefile
@@ -8,7 +8,7 @@
 
 HD_CXXFLAGS		= ${HD_STD_CXXFLAGS}
 
-HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahtime -lahmission \
+HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahtime -lahmission -lahcaldb \
 			  -lahapp -l${AHFITS} -lahmath -l${AHGEN} -l${AHLOG} \
 			  ${HD_STD_LIBS} ${F77LIBS4C}
 
--- hitomixrism/common/cal/lib/sxsrsl/src/Makefile
+++ hitomixrism/common/cal/lib/sxsrsl/src/Makefile
@@ -8,7 +8,7 @@
 
 HD_CXXFLAGS		= ${HD_STD_CXXFLAGS}
 
-HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahapp -l${AHFITS} -lahmath \
+HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahapp -l${AHFITS} -lahmath -lahmission \
 			  -l${AHGEN} -l${AHLOG} ${HD_STD_LIBS} ${F77LIBS4C}
 
 HD_INSTALL_LIBRARIES	= ${HD_LIBRARY_ROOT}
--- hitomixrism/common/mission/tasklib/timeassign/src/Makefile
+++ hitomixrism/common/mission/tasklib/timeassign/src/Makefile
@@ -8,7 +8,7 @@
 
 HD_CXXFLAGS		= ${HD_STD_CXXFLAGS}
 
-HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahtime -lahmission \
+HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahtime -lahmission -lahcaldb \
 			  -lahapp -l${AHFITS} -lahmath -l${AHGEN} -l${AHLOG} \
 			  ${HD_STD_LIBS} ${F77LIBS4C}
 
--- hitomixrism/common/mission/tasklib/trendtemp/src/Makefile
+++ hitomixrism/common/mission/tasklib/trendtemp/src/Makefile
@@ -8,7 +8,7 @@
 
 HD_CXXFLAGS		= ${HD_STD_CXXFLAGS}
 
-HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahcaldb -lahapp -l${AHFITS} -lahmath -l${AHGEN} -l${AHLOG} \
+HD_SHLIB_LIBS		= ${HD_LFLAGS} -lahcaldb -lahapp -l${AHFITS} -lahmath -lahmission -lahtime -l${AHGEN} -l${AHLOG} \
 			  ${HD_STD_LIBS} ${F77LIBS4C}
 
 HD_INSTALL_LIBRARIES	= ${HD_LIBRARY_ROOT}
